### PR TITLE
Add list operation example

### PR DESCRIPTION
**Pull request by koertho**
This PR adds an example for list operations to the dca docs.

### CODE REVIEW

**Review by fritzmg — CHANGES_REQUESTED** [review]
Unfortunately I don't think this is a good example. This might give the false impression that this prevents non-admins from deleting records. However it only hides the button - and does not prevent the actual action.

**Comment by koertho** [comment]
Ok, I changed the example to a custom button and added additional explanation.
I also enhanced the code with Backend class instead of Controller class and the usage of sprintf.

File: docs/dev/reference/dca/callbacks.md [hunk 1]
```diff
@@ -599,6 +599,64 @@ an additional command check via load_callback).
 **return:** `string` HTML for the button
 {{% /expand %}}
 
+{{% expand "Example" %}}
+
+This example hide a custom operation button if the user is not allowed to use it.    
+
+Attention: this won't disable the operation itself, it only hides the button!
+To disable the operation, you need to check for the permission additional before it is execution, for example in the operation backend module code.
```
Comment (fritzmg) [inline docs/dev/reference/dca/callbacks.md]:
```suggestion
Attention: this won't disable the operation itself, it only hides the button!
To disable the operation, you need to check for the permission additionally 
before its execution, for example in the operation code or a `config.onload` callback.
```

**Review by fritzmg — CHANGES_REQUESTED** [review]

**Review by fritzmg — APPROVED** [review]

**Comment by fritzmg** [comment]
Thank you @koertho :)

**Pull request merged**